test/parser/mixins.spec.js

```diff
@@ -25,6 +25,14 @@ describe('Parser', () => {
             expect(root.first.type).to.eql('atrule');
         });
 
+        it('should parse parametric mixins', () => {
+            const root = parse('.foo(@bar) { color: @bar;}', {mixinsAsAtRules: true, innerMixinsAsRules: true});
+
+            expect(root.first.type).to.eql('atrule');
+            expect(root.first.params[0].type).to.eql('atrule');
```

Comment:
We would like to handle many LESS features: params, all types of mixins, 'rest' params, etc. At the same time PostCSS supports a few [syntax constructions](https://github.com/postcss/postcss/blob/master/docs/api.md): Declarations, AtRules, Rules, Comments. So I try to parse LESS within these limits and the things like 'lessType' are hacks to provide some additional capabilities for 3rd-party libraries.